packages/evm-adapter/src/utils.ts

```diff
@@ -1,4 +1,21 @@
+import { BigNumber, ContractInterface } from 'ethers';
+import { helio, helioEth } from './abi';
 import { ChainId, ContractAddress } from './constants';
+import { PaymentRequest } from './types';
 
 export const getContractAddress = (chainIdNr: number): string | undefined =>
   chainIdNr in ChainId ? (<any>ContractAddress)[ChainId[chainIdNr]] : undefined;
+
+export const getFeesAndAddresses = (req: PaymentRequest) => {
```

Comment:
can this get a return type? 
its used in places typed as `any[]` - useful to have at least some kind of type checks here..